RELEASE_NOTES.md

```diff
@@ -3,6 +3,7 @@
 ### 0.6.0  Multi-select (WIP)
 - Changes UI to use new dark theme
 - Adds "Open File and Exit" command
+- #92 Adds per-folder storage of sort settings (Thanks @jilleJr)
```

Comment:
@mattstermiller  These are a little out of order right now. Should I move the first "Changes" line to below the "Adds" lines, or move the "Adds" note for this PR to at the end?

Reply:
I try to arrange release notes by kind in this order:
1. major additions/changes
2. new features
3. minor improvement or changes
4. bug fixes

As such, it's in order as the latest new feature.